tidb-cloud/tidb-cloud-password-authentication.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Password Authentication
+summary: Learn how to manage passwords and enable multi-factor authentication (MFA) in the TiDB Cloud console.
+---
+
+# Password Authentication
+
+This document describes how to manage passwords and enable multi-factor authentication (MFA) in the TiDB Cloud console. The document is only applicable to users who [sign up](https://tidbcloud.com/free-trial) for TiDB Cloud with emails and passwords.
+
+## Sign up
+
+You can either [sign up](https://tidbcloud.com/free-trial) for TiDB Cloud with email and password, or choose your Google Workspace or GitHub account for single sign-on (SSO) to TiDB Cloud.
+
+- If you sign up for TiDB Cloud with email and password, you can manage your password according to this document.
+- If you choose Google SSO or GitHub SSO to TiDB Cloud, your password is managed by Google or GitHub and you cannot change it using the TiDB Cloud console.
+
+To sign up for a TiDB Cloud account with email and password, take the following steps:
+
+1. Go to the TiDB Cloud [sign up](https://tidbcloud.com/free-trial) page and fill in the registration information.
+
+2. Read Privacy Policy and Services Agreement, and then select **I agree to the Privacy Policy and Services Agreement**.
+
+3. Click **Sign up**.
+
+You will receive a verification email for TiDB Cloud. To complete the whole registration process, check your email box and confirm the registration.
+
+## Sign in or sign out
+
+### Sign in
+
+To log into TiDB Cloud using email and password, take the following steps:
+
+1. Go to the TiDB Cloud [login](https://tidbcloud.com/) page.
+
+2. Fill in your email and password.
+
+3. Click **Sign In**.
+
+If the login is successful, you will be directed to the TiDB Cloud console.
+
+### Sign out
+
+In the upper-right corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-account-settings" /> **Account** and select **Logout**.
+
+## Password policy
+
+TiDB Cloud sets a default password policy for registered users. If your password does not meet the policy, you will get a prompt when you set the password.
+
+The default password policy is as follows:
+
+- At least 8 characters in length.
+- At least 1 uppercase letter (A-Z).
+- At least 1 lowercase letter (a-z).
+- At least 1 number (0-9).
+- A new password must not be the same as any of the previous four passwords.
+
+## Reset a password
+
+> **Note:**
+>
+> This section is only applicable to TiDB Cloud registration with email and password. If you sign up for TiDB Cloud with Google SSO or GitHub SSO, your password is managed by Google or GitHub and you cannot change it using the TiDB Cloud console.
```

Comment:
Maybe we can remove this note because this part has been clarified in [line 14 and line 15](https://github.com/pingcap/docs/pull/11614/files#diff-3e0c486475f106a4567665c838f181980bb54e30c1e0831fab4c11018da79fedR14-R15)

Reply:
Just in case some docs might contain a cross-link to this section directly, it is still worth keeping this note here.